Makefile

```diff
@@ -226,7 +228,7 @@ endif
 
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | $(KUBECTL) apply -f -
+	$(KUSTOMIZE) build config/crd | $(KUBECTL) apply --server-side --force-conflicts -f -
```

Comment:
Will --force-conflicts  cause other problems? Should be careful here, prefer to keep the server-side only.